packages/zero-next-plugin/src/index.ts

```diff
@@ -3,7 +3,7 @@ import type { NextConfig } from 'next';
 import { findPagesDir } from 'next/dist/lib/find-pages-dir';
 import {
   webpack as zeroWebpackPlugin,
-  PluginOptions as BaseZeroPluginConfig,
+  type PluginOptions as BaseZeroPluginConfig,
```

Comment:
Even though I agree with using explicit `type`, we should mention this in the core meeting to set the consistency.

Reply:
But we can't do this with the current version of TypeScript supported https://mui.com/material-ui/getting-started/supported-platforms/#typescript, e.g. #38296. 

I think we need to change this import. I don't know if #40996 would be enough.

Reply:
I believe we can write this in our source code but it should not end up in the generated `d.ts` files. This is exactly what is happening right now. If you look at the output, it's like this -

```ts
import { PluginOptions } from '@mui/zero-unplugin';

type ZeroPluginConfig = PluginOptions & {
    asyncResolve?: (what: string) => string | null;
};
```

So there is no `type` import here.